web/explorer/src/components/RuleMatchesTable.vue

```diff
@@ -0,0 +1,289 @@
+            <template #item="{ item, props }">
+                <a v-ripple v-bind="props.action" :href="item.url" :target="item.target">
+                    <span v-if="item.icon !== 'vt-icon'" :class="item.icon" />
+                    <VTIcon v-else-if="item.icon === 'vt-icon'" />
+                    <span>{{ item.label }}</span>
+                </a>
+            </template>
+        </ContextMenu>
+
+        <Toast />
+
+        <Dialog v-model:visible="sourceDialogVisible" :style="{ width: '50vw' }">
+            <highlightjs autodetect :code="currentSource" />
+        </Dialog>
+    </div>
+</template>
+
+<script setup>
+import "highlight.js/styles/stackoverflow-light.css";
+
+import { ref, onMounted, computed } from "vue";
+import TreeTable from "primevue/treetable";
+import InputText from "primevue/inputtext";
+import Dialog from "primevue/dialog";
+import Column from "primevue/column";
+import IconField from "primevue/iconfield";
+import InputIcon from "primevue/inputicon";
+import ContextMenu from "primevue/contextmenu";
+
+import RuleColumn from "./columns/RuleColumn.vue";
+import VTIcon from "./misc/VTIcon.vue";
+
+import { parseRules } from "../utils/rdocParser";
+import { createMBCHref, createATTACKHref } from "../utils/urlHelpers";
+
+const props = defineProps({
+    data: {
+        type: Object,
+        required: true
+    },
+    showLibraryRules: {
+        type: Boolean,
+        default: false
+    }
+});
+
+const treeData = ref([]);
+const filters = ref({});
+const filterMode = ref("lenient");
+const sourceDialogVisible = ref(false);
+const currentSource = ref("");
+const expandedKeys = ref({});
+
+const menu = ref();
+const selectedNode = ref({});
+
+const contextMenuItems = computed(() => [
+    {
+        label: "View source",
+        icon: "pi pi-eye",
+        command: () => {
+            showSource(selectedNode.value.data.source);
+        }
+    },
+    {
+        label: "View rule in capa-rules",
+        icon: "pi pi-external-link",
+        target: "_blank",
+        url: selectedNode.value.url
+    },
+    {
+        label: "Lookup rule in VirusTotal",
+        icon: "vt-icon",
+        target: "_blank",
+        url: selectedNode.value.vturl
+    }
+]);
+
+const onRightClick = (event, instance) => {
+    if (instance.node.data.source) {
+        selectedNode.value = instance.node;
+        // contrust capa-rules url
+        selectedNode.value.url = `https://github.com/mandiant/capa-rules/blob/master/${instance.node.data.namespace || "lib"}/${instance.node.data.name.toLowerCase().replace(/\s+/g, "-")}.yml`;
+        // construct VirusTotal deep link
+        const behaviourSignature = `behaviour_signature:"${instance.node.data.name}"`;
+        selectedNode.value.vturl = `https://www.virustotal.com/gui/search/${encodeURIComponent(behaviourSignature)}/files`;
+
+        menu.value.show(event);
+    }
+};
+
+/*
+ * Expand node on click
+ */
+const onNodeSelect = (node) => {
```

Comment:
can we set the default mouse cursor for these items (not the select-text one)?

Reply:
Should be good now :)